crates/turbopack-node/src/lib.rs

```diff
@@ -480,13 +444,72 @@ enum RenderProxyIncomingMessage {
     Error(StructuredError),
 }
 
+#[derive(Serialize)]
+#[serde(tag = "type", rename_all = "camelCase")]
+enum EvalJavaScriptOutgoingMessage {
+    LoadNextConfig { path: String },
```

Comment:
```suggestion
    LoadConfig { path: String },
```

Reply:
let's keep the rpc name as `LoadNextConfig`,  the follow-up will add more different types of config reading logic

Reply:
resolved